### DISCUSSION TITLE
Atom Mapping Index goes wrong for a reaction

### DISCUSSION THREAD

**Discussion by tsnguyen05** [rdkit/rdkit#4395] [Q&A]

**Comment by greglandrum** [comment]
@tsnguyen05 in order to help us try and answer your question, please provide a specific example showing what you would like to provide as input and what you expect the output to be.

**Reply by tsnguyen05** [reply]
Hello @greglandrum,

May I ask whether the SMART string of the product from an atom-mapping example went through the pre-processing before being used for ReactionFromSmarts?

In case there is no way to fix this, I wonder if I can request your snippet to manually do atom-map myself (https://github.com/rdkit/rdkit/issues/2325#issuecomment-472265760). Furthermore, could you please share a written procedure of an atom mapping on rdkit so that I can understand it deeper? I tried to look for it but could not find it anywhere.

**Reply by greglandrum** [reply]
> 
> May I ask whether the SMART string of the product from an atom-mapping example went through the pre-processing before being used for ReactionFromSmarts?

I'm sorry that I don't understand the question. Could you please try rephrasing it or provide an example?
